server/src/main/java/io/seata/server/storage/redis/lock/RedisLocker.java

```diff
@@ -200,48 +217,28 @@ public boolean releaseLock(String xid, Long branchId) {
     @Override
     public boolean isLockable(List<RowLock> rowLocks) {
         if (CollectionUtils.isEmpty(rowLocks)) {
-            // no lock
             return true;
         }
         try (Jedis jedis = JedisPooledFactory.getJedisInstance()) {
             List<LockDO> locks = convertToLockDO(rowLocks);
             Set<String> lockKeys = new HashSet<>();
             for (LockDO rowlock : locks) {
-                lockKeys.add(getLockKey(rowlock.getRowKey()));
+                lockKeys.add(buildLockKey(rowlock.getRowKey()));
             }
-            List<String> rowlockJsons = jedis.mget(lockKeys.toArray(new String[0]));
+
             String xid = rowLocks.get(0).getXid();
-            for (String rowlockJson : rowlockJsons) {
-                if (!StringUtils.isEmpty(rowlockJson)) {
-                    LockDO lock = JSON.parseObject(rowlockJson, LockDO.class);
-                    if (lock != null && !Objects.equals(lock.getXid(), xid)) {
-                        return false;
-                    }
-                }
-            }
+            Pipeline pipeline = jedis.pipelined();
+            lockKeys.stream().forEach(key -> pipeline.hget(key, XID));
+            List<String> existedXids = (List<String>) (List) pipeline.syncAndReturnAll();
+            return existedXids.stream().allMatch(existedXid -> existedXid == null || xid.equals(existedXid));
         }
-        return true;
-    }
-
-    private Set<String> lRange(Jedis jedis, String key) {
-        Set<String> keys = new HashSet<>();
-        List<String> redisLockJson;
-        int start = 0;
-        int stop = logQueryLimit;
-        do {
-            redisLockJson = jedis.lrange(key, start, stop);
-            keys.addAll(redisLockJson);
-            start = keys.size();
-            stop = start + logQueryLimit;
-        } while (CollectionUtils.isNotEmpty(redisLockJson));
-        return keys;
     }
 
-    private String getXidLockKey(String xid) {
+    private String buildXidLockKey(String xid) {
         return DEFAULT_REDIS_SEATA_LOCK_XID_PREFIX + xid;
     }
 
-    private String getLockKey(String rowKey) {
+    private String buildLockKey(String rowKey) {
         return DEFAULT_REDIS_SEATA_LOCK_PREFIX + rowKey;
```

Comment:
Change the names of these two keys for structure error

Reply:
> Change the names of these two keys for structure error

finished